projects/pastanaga-angular/src/lib/styles/_layout.scss

```diff
@@ -69,3 +69,24 @@
         width: $width-formula-panel-plus-wide-large;
     }
 }
+
+.pa-scrollable-container-with-fixed-footer {
+    margin-bottom: rhythm(20);
+}
+
+.pa-fixed-footer-compact {
+    position: fixed;
+    bottom: 0;
+    padding: rhythm(6) 0;
+    height: rhythm(6) * 3;
```

Comment:
why your height isn't `rhythm(20)`? I think it should be the same as what you have as `margin-bottom` in `.pa-scrollable-container-with-fixed-footer`, so having a variable for it would probably be a good idea.